lib/server.js

```diff
@@ -19,7 +27,9 @@ module.exports = async () => {
   const initialDir = await getInitialDir();
   const initialPath = await getInitialPath();
 
-  app.use(favicon(path.join(__dirname, 'favicon.ico')));
+  const customFavicon = await getCustomFavicon();
+  const faviconPath = customFavicon || path.join(__dirname, 'favicon.ico');
```

Comment:
There is logic for the favicon in three locations (here, util, and config). Could you please put it all in one location please? I guess the config module makes sense.

Reply:
I've moved the favicon path resolution to config.js. I initially placed the default one to server, as in the case of moving `config.js` elsewhere then `server.js` would then break the default favicon behavior. I guess premature optimization or something. Anyway I've moved the logic to config now, and I've also added a line to docs (README) to document this for reveal-md users.